perllib/FixMyStreet/Integrations/ExorRDI.pm

```diff
@@ -37,13 +37,18 @@ has user => (
     },
 );
 
+my $now = DateTime->now(
```

Comment:
Has to be in the construct function, I'm afraid, otherwise the web interface will only get it sent once on server startup :)